src/lib/undux/SimpleStore.js

```diff
@@ -87,7 +87,7 @@ const initialState: State = {
   currentFeed: null,
   addWebApp: {
     show: false,
-    showMenuItem: false,
+    showInMainMenu: false,
```

Comment:
a good name describes what it does:
showAddWebAppDialog

Reply:
> Add Icon is hidden on iPad:
> <img alt="Screenshot 2020-01-08 at 12 43 17" width="676" src="https://user-images.githubusercontent.com/36693645/71971989-74e32780-3214-11ea-8e70-89de7c14557d.png">
> 
> we have a problem with IPAD. because Ipad uses PadOs and we can't detect is it Ipad or desktop. one of the solutions to detect is comparing width/height.
> https://stackoverflow.com/questions/57776001/how-to-detect-ipad-pro-as-ipad-using-javascript
> what do u think @sirpy do we need implement detecting IPad by width/height or maybe u know another way?

@AlexeyKosinski not important, we are working on a native app

Reply:
addWebApp.showAddWebAppDialog

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1144/commits/f45152a995064e1e17b2425cb79e854ee6d9ef50